src/forms/BenefitsTable.js

```diff
@@ -32,11 +32,11 @@ const BenefitsTable = function ( props ) {
   // of just using current value.
   currentClient.future.earned = currentClient.current.earned;
 
-  var SNAPBenefitCurrent  = client.current.hasSnap ? Math.round( getSNAPBenefits( client, 'current' ).benefitValue * 12 ) : 0,
-      SNAPBenefitFuture   = client.current.hasSnap ? Math.round( getSNAPBenefits( client, 'future' ).benefitValue * 12 ) : 0,
+  var SNAPBenefitCurrent  = client.current.hasSnap ? Math.round( getSNAPBenefits( client, 'current' ) * 12 ) : 0,
+      SNAPBenefitFuture   = client.current.hasSnap ? Math.round( getSNAPBenefits( client, 'future' ) * 12 ) : 0,
       SNAPDiff            = SNAPBenefitFuture - SNAPBenefitCurrent,
-      sec8BenefitCurrent  = client.current.hasHousing ? Math.round( getHousingBenefit( client ).benefitValue * 12 ) : 0,
-      sec8BenefitFuture   = client.current.hasHousing ? Math.round( getHousingBenefit( client ).benefitValue * 12 ) : 0,
+      sec8BenefitCurrent  = client.current.hasHousing ? Math.round( getHousingBenefit( client ) * 12 ) : 0,
+      sec8BenefitFuture   = client.current.hasHousing ? Math.round( getHousingBenefit( client ) * 12 ) : 0,
```

Comment:
Now that `getSNAPBenefits()` and `getHousingBenefit()` return numbers, these conditionals could be moved into those functions.

Reply:
Hmm, interesting thought. I can see the perspective that if a person isn't receiving a benefit, they're getting 0 for that benefit, but I could also see that if someone isn't receiving a benefit, we shouldn't be trying to calculate it in the first place. We should keep it mind as we see how things pan out.